<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>1109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IAP升级</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1584</width>
     <height>934</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTextBrowser" name="textBrowser">
            <property name="minimumSize">
             <size>
              <width>1200</width>
              <height>598</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QWidget" name="usartview" native="true">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>600</height>
               </size>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>通讯方式</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="communication">
                   <item>
                    <property name="text">
                     <string>以太网</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>串口</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>波特率</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="bundrate">
                   <item>
                    <property name="text">
                     <string>115200</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>停止位</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="stopbits">
                   <item>
                    <property name="text">
                     <string>1</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>2</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>3</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>校验位</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="parity">
                   <item>
                    <property name="text">
                     <string>无校验</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>奇校验</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>偶校验</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>数据位</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="databits">
                   <item>
                    <property name="text">
                     <string>8</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>9</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>串口号</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QComboBox" name="serialport"/>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>APP选择</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBox">
                 <item>
                  <property name="text">
                   <string>1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="linkusart_3">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>连接串口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dislinkusart">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>断开串口</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clear">
              <property name="text">
               <string>清空</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>服务器地址</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_7">
              <property name="text">
               <string>192.168.10.122</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>端口号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="text">
               <string>5001</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_7">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QPushButton" name="linkserver">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>连接服务器</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dislinkserver">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>断开服务器</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>1580</width>
          <height>100</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="Sendfile">
           <property name="text">
            <string>发送升级包</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Openfile">
           <property name="text">
            <string>将bin文件构建为升级包</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>信息发送</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="senddata">
         <property name="text">
          <string>发送</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>textBrowser</tabstop>
  <tabstop>Sendfile</tabstop>
  <tabstop>parity</tabstop>
  <tabstop>stopbits</tabstop>
  <tabstop>serialport</tabstop>
  <tabstop>bundrate</tabstop>
  <tabstop>databits</tabstop>
  <tabstop>communication</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
